run_manylinux_aarch64_build_publish.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+__heredoc__="""
+notes:
+    Manylinux repo: https://github.com/pypa/manylinux 
+    Win + Osx repo: https://github.com/mavlink/MAVSDK-Python
+"""
+
+
+DOCKER_IMAGE=${DOCKER_IMAGE:="quay.io/pypa/manylinux2014_aarch64:latest"}
```

Comment:
I believe the existing publish script will work as long as you have pre-built the right images. It includes a glob pattern to capture pre-built binary wheels, so that should just capture the aarch64 ones.

Reply:
> I believe the existing publish script will work as long as you have pre-built the right images. It includes a glob pattern to capture pre-built binary wheels, so that should just capture the aarch64 ones.

Done. Please share your feedback regarding the changes.